.test-infra/jenkins/dependency_check/generate_report.sh

```diff
@@ -39,7 +39,7 @@ REPORT_DESCRIPTION="
 
 
 # Virtualenv for the rest of the script to run setup
-/usr/bin/virtualenv dependency/check
+virtualenv dependency/check
```

Comment:
We don't have to clean at beginning. The virtualenv was installed by pip3 so that the default version will be python3.5. That's why we change those py files to make them compatible with the environment.

Reply:
sg